kits/support-to-engineering-bug-bridge/README.md

```
@@ -0,0 +1,181 @@
+| Distinct Accounts Affected | Severity |
+|---|---|
+| 1 | P4 — Minor |
+| 2–3 | P3 — Moderate |
+| 4–6 | P2 — High |
+| 7+ | P1 — Critical |
+
+---
+
+## Setup
+
+### Prerequisites
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)